ext/opentelemetry-ext-otlp/src/opentelemetry/ext/otlp/__init__.py

```diff
@@ -30,10 +30,13 @@
 
     from opentelemetry import trace
     from opentelemetry.ext.otlp.trace_exporter import OTLPSpanExporter
+    from opentelemetry.sdk.resources import Resource
     from opentelemetry.sdk.trace import TracerProvider
     from opentelemetry.sdk.trace.export import BatchExportSpanProcessor
 
-    trace.set_tracer_provider(TracerProvider())
+    trace.set_tracer_provider(TracerProvider(resource=Resource(labels={
+        "service.name": "service"
+    }))))
```

Comment:
i think there's one too many `)`

Could you add a comment above this line explaining that some backends like jaeger requires resources with the service.name being set? Otherwise I can imagine people who wouldn't set it because they didn't think it was needed and run into the same issue you have.

Reply:
Got it, made a small decomposition to visually demarcate the code and left some comment to describe the need for resources